src/main/java/net/fortuna/ical4j/model/DateTime.java

```diff
@@ -410,7 +410,8 @@ private void setTime(final String value, final DateFormat format,
 			final java.util.TimeZone tz) throws ParseException {
 
 		if (tz != null) {
-			format.setTimeZone(tz);
```

Comment:
The reason this works is that it converts the ical4j timezone to a standard java.util.TimeZone definition. Unfortunately this will only work for standard timezone identifiers, and it doesnt actually support use of custom timezone definitions.

I will use the test as a basis for fixing the observance implementation, many thanks.